doc/api/modules.markdown

```diff
@@ -346,6 +346,12 @@ If this was in a folder at `./some-library`, then
 
 This is the extent of Node.js's awareness of package.json files.
 
+Note: If the file specified by the `"main"` entry of `package.json` is missing and can not be resolved, node will report the entire module as missing with the default error:
```

Comment:
can you wrap this at 80 characters. you will notice this is the longest line in the file